_rules/html-page-lang-b5c3f8.md

```diff
@@ -29,11 +29,11 @@ This rule applies to any [document element](https://dom.spec.whatwg.org/#documen
 
 ## Expectation
 
-Each test target has a `lang` attribute that is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
+Each test target has a `lang` attribute that is neither empty (`""`) nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
```

Comment:
Why are you changing this?

Reply:
I think, technically this is better. 

("") 
could mean 
attribute="value" lang= attribute "value"

While 
(`""`) 
explicitly means
attribute="value" lang="" attribute "value"

But it is a really small difference.